back-office/international/translations/translations.md

```diff
@@ -1,12 +1,12 @@
 # Translations
 
-## Conceptual vocabulary
+#### Conceptual vocabulary
 
-- Wordings: original texts found throughout the software. They are in English but may be translated _to_ English for better localization.
-- Translations: wordings translated to a given language.
-- Translation domain: named logic group of wordings. Any given wording is attached to one and only one translation domain. Two identical wordings attached to two different translation domains are considered as two different wordings.
+- **Wordings:** original texts found throughout the software. They are in English but may be translated _to_ English for better localization.
+- **Translations:** wordings translated to a given language.
+- **Translation domain:** named logic group of wordings. Any given wording is attached to one and only one translation domain. Two identical wordings attached to two different translation domains are considered as two different wordings.
 - Translation catalogue: A collection of translation domains containing translations for a given language, stored physically in a support like files or on a database.
```

Comment:
```suggestion
- Translation catalog: A collection of translation domains containing translations for a given language, stored physically in a support like files or on a database.
```

Reply:
Seems that both syntaxes are acceptable

Reply:
maybe that's because we use `catalog` in other places?

Reply:
In the software, we tend to use 'catalog' from what I saw on Crowdin. Both 'catalog' and 'catalogue' are acceptable. The first option is more popular in American English. But Pablo told me we use 'Catalogue' here because that's the one chosen by Simfony's implementation.

Reply:
That's right, Symfony writes "catalog" in the docs and "catalogue" in the code.